test_suite/generic/helpers.py

```diff
@@ -1,10 +1,10 @@
-from test_suite import conftest
+import os
+import json
 
 
-def instance_data(host):
-    values_to_find = [host.backend.hostname]
-    values_to_find.extend(host.addr(host.backend.hostname).ipv4_addresses)
-
-    return conftest.__get_instance_data_from_json(
-        key_to_find='address', values_to_find=values_to_find
-    )
+def __get_instance_data_from_json(key_to_find, values_to_find):
```

Comment:
```suggestion
def __get_instance_data_from_json(instances_json_path, key_to_find, values_to_find):
    with open(instances_json_path, 'r') as f:
```

Reply:
Oh ok I didn't notice to this line "with open(instances_json_path, 'r') as f:"  this is why I didn't understand what's the benefit from this :) cause then it showed this one : "with open(os.environ['CIV_INSTANCES_JSON'], 'r') as f:" as it supposed to stay and not removed.

In general I agree about this, but since it is CIV Project so I don't think other instance json file should be handled here.

But to make it modular as you suggested I would create a configuration veriable in the helpers file since it's permenent variable for this CIV project and it's only 'CIV_INSTANCES_JSON', instead of adding this each time you call this function.

INSTANCES_JSON_PATH = <path/CIV_INSTANCES_JSON>
def __get_instance_data_from_json(path = 'INSTANCES_JSON_PATH', key_to_find, values_to_find):
        with open(instances_json_path, 'r') as f:
            instances_json_data = json.load(f)
            for instance in instances_json_data.values():
                if key_to_find in instance.keys() and instance[key_to_find] in values_to_find:
                    return instance